kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -121,15 +121,15 @@ define([
             bus = runtime.bus().makeChannelBus({
                 description: 'A file path widget',
             });
-        let container, ui;
+        let container, ui, appSpec;
```

Comment:
Capture the actual `Spec` object here (that has generalized validators and such). And don't store it in the model. Not sure why it was needed there anyway.

Reply:
This affects a few other function signatures, too, as it was unpacked from the model and passed around. Now, it's just part of the object.

Reply:
Sensible!